### PR TITLE
Implement simple version of `getaccount` endpoint

### LINKED ISSUE #692: Implement quick version of `getaccount` endpoint
Include all the information that we can easily get from the current backend. This should include
* `asset_types: Vec<AssetDefinition>` for viewing and freezing keys
* `address` for all key types (will be one of `UserAddress`, `AuditorPubKey`, or `FreezerPubKey`)
* `records` for all key types (will not include partially viewable records for viewing keys)
Each record should include
* `asset`
* `amount`
* `uid`
In addition, each record associated with a viewing or freezing key should contain the `address` of the owner.

### PR DESCRIPTION
**Pull request by jbearer**
Closes #692

### CODE REVIEW

File: wallet/api/api.toml [hunk 1]
```diff
@@ -92,6 +92,23 @@ DOC = """
...
 
+[route.getaccount]
+PATH = ["getaccount/:address"]
+":address" = "TaggedBase64"
+DOC = """
+Get information about an account.
+
+`:address` can be an account address, or a viewing, freezing, or sending public key. In any case, the result contains at least the following fields:
+* `"records"` - a list of records related to this account. For addresses and sending public keys, it is a list of owned records. For viewing keys it is a list of viewable records, and for freezing keys, a list of freezable records.
+`:assets` - assets related to this account, indexed by code. Each asset is an `AssetInfo` structure, as returned by `getinfo/asset`. For addresses and sending public keys, this is just the collection of assets currently owned by the account. For viewing and freezing keys, it also includes all assets known to the wallet which are viewable or freezable using that key. It is still guaranteed to contain the asset type of every record in `"record"`.
```
Comment (shenkeyao) [inline wallet/api/api.toml:103]: Incomplete sentence?

File: wallet/src/routes.rs [hunk 2]
```diff
@@ -834,6 +809,105 @@ pub async fn get_last_keystore(path: &Option<PathBuf>) -> Result<Option<PathBuf>
...
+                Some((record.clone().into(), record.ro.asset_def.code))
+            } else {
+                None
+            }
+        })
+        .unzip();
+
+    Account {
+        records,
+        // `assets` is only applicable for viewing and freezing accounts.
```
Comment (shenkeyao) [inline wallet/src/routes.rs:837]: Does this mean `assets` can only be viewed or frozen, but not spent? Then why does it belong to the sending account?

**Review by shenkeyao — CHANGES_REQUESTED** [review]
The code looks good. Just need some comment changes.

File: wallet/src/routes.rs (hunk 2 above)
Comment (jbearer) [reply wallet/src/routes.rs:837]: Oh I should have removed this

**Review by shenkeyao — APPROVED** [review]

**Pull request merged**